tests/test_models.py

```diff
@@ -1,38 +1,186 @@
         """ This runs once before the entire test suite """
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        Wishlist.init_db(app)
 
     @classmethod
     def tearDownClass(cls):
         """ This runs once after the entire test suite """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
+        #db.drop_all() #Drops all tables if needed for updating schema
 
     def setUp(self):
         """ This runs before each test """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
 
     def tearDown(self):
         """ This runs after each test """
+        db.session.remove()
 
     ######################################################################
     #  T E S T   C A S E S
     ######################################################################
 
-    def test_example_replace_this(self):
-        """ It should always be true """
-        self.assertTrue(True)
+    def test_create_a_wishlist(self):
+        """It should Create a Wishlist and assert that it exists"""
+        fake_wishlist = WishlistFactory()
+        # pylint: disable=unexpected-keyword-arg
+        wishlist = Wishlist(
+            customer_id=fake_wishlist.customer_id,
+            wishlist_name=fake_wishlist.wishlist_name,
+            created_date=fake_wishlist.created_date,
+        )
+        self.assertIsNotNone(wishlist)
+        self.assertEqual(wishlist.wishlist_id, None)
+        self.assertEqual(wishlist.customer_id, fake_wishlist.customer_id)
+        self.assertEqual(wishlist.wishlist_name, fake_wishlist.wishlist_name)
+        self.assertEqual(wishlist.created_date, fake_wishlist.created_date)
+
+    def test_add_a_wishlist(self):
+        """It should Create a Wishlist and add it to the database"""
+        wishlists = Wishlist.all()
+        self.assertEqual(wishlists, [])
+        wishlist = WishlistFactory()
+        wishlist.create()
+        # Assert that it was assigned an id and shows up in the database
+        self.assertIsNotNone(wishlist.wishlist_id)
+        wishlists = Wishlist.all()
+        self.assertEqual(len(wishlists), 1)
+
+    def test_read_wishlist(self):
+        """It should Read a Wishlist"""
+        wishlist = WishlistFactory()
+        wishlist.create()
+
+        # Read it back
+        found_wishlist = Wishlist.find(wishlist.wishlist_id)
+        self.assertEqual(found_wishlist.wishlist_id, wishlist.wishlist_id)
+        self.assertEqual(found_wishlist.customer_id, wishlist.customer_id)
+        self.assertEqual(found_wishlist.wishlist_name, wishlist.wishlist_name)
+        self.assertEqual(found_wishlist.created_date, wishlist.created_date)
+
+    def test_update_wishlist(self):
+        """It should Update a Wishlist"""
+        wishlist = WishlistFactory(wishlist_name="change name")
```

Comment:
Might be slightly better to store "change name" in a variable and reference in both places; the WishlistFactory constructor and the assert statement.  (Very minor point.)

Reply:
Got it, will make it a variable for clarity.